.gitignore

```diff
@@ -6,3 +6,19 @@ build/
 
 # editor config
 .vscode/
+
+# Xcode
+*.pbxuser
```

Comment:
These files contain user settings for Xcode. This particular one saves positions of windows. This should not be in the repo.